### DISCUSSION TITLE
Fabric.js Google Group has become unmoderated

### DISCUSSION THREAD

**Discussion by crockwave** [fabricjs/fabric.js#9566] [General]
The Fabric.js Google Group has become unmoderated, filled with spam, and now is listed with a Banned Content Warning.

Should that group be resurrected, since it has much useful information in it?  Should a new Google group be created?

The Fabric.js Support page references that group

I can help moderate that group, if it would benefit

**Comment by ShaMan123** [comment]
IMO it should be closed and a pinned message should be added saying that all dicussions should occur in the repo's discussions.
Having everything in the repo is very powerful and easy to maintain.
But it is not a decision to make by myself @asturur 
Regardless, you are most welcome to join the effort as a collaborator.

**Comment by asturur** [comment]
I do not have control on that group, i didn't create it, i do not know how to close it, i did try in the past.
